src/specs/components/language-list/demo/data--overlay.js

```diff
@@ -0,0 +1,187 @@
+module.exports = {
+  title: 'Select your language',
+  items: [
+    {
+      label: 'български',
+      path: '/example',
+      aria_label: '',
```

Comment:
In a previous pull request I removed empty value properties. 
I don't mind having such. I suggest however we keep consistency.

Reply:
sure, i'll remove them, my bad ;)